Assets/Tests/InputSystem/Plugins/InputForUITests.cs

```diff
@@ -118,7 +101,7 @@ public void TouchIsPressedAndMouseEventsAreIgnored()
         Move(mouse.position, new Vector2(100f, 0.5f));
         Update();
 
-        Assert.IsTrue(m_InputForUIEvents.Count == 1);
+        Assert.AreEqual(1, m_InputForUIEvents.Count);
```

Comment:
I was not part of establishing the convention so do not now why, but Assert.That(m_InputForUIEvents.Count, Is.EqualTo(1)); seems preferred instead. Likely due to not mixing up expected and actual argument order...

Reply:
Oh, interesting - I assumed everyone was using Rider which shows the arguments:
<img width="480" alt="image" src="https://github.com/Unity-Technologies/InputSystem/assets/102745393/be060adc-9383-4e59-802b-22aa9c531d60">
Though I guess github lacks that 😄